README.md

```diff
@@ -76,6 +61,7 @@ MetadataJsonLint.options.strict_license = false
 * `--[no-]strict-dependencies`: Whether to fail if module version dependencies are open-ended. Defaults to `false`.
 * `--[no-]strict-license`: Whether to fail on strict license check. Defaults to `true`.
 * `--[no-]fail-on-warnings`: Whether to fail on warnings. Defaults to `true`.
+* `--[no-]strict-puppet-version`: Whether to fail if Puppet version requirements are open-ended and no longer supported. Defaults to `false`.
```

Comment:
Doesn't open-ended mean there aren't limits one side? Should this be `open-ended or no longer`?

Reply:
It should be `open-ended or no longer supported`. I'll double check the code, but I'm pretty sure the `open-ended` check is specifically checking for an open-ended upper bound. It doesn't make much sense to check for open endedness on the lower bound. 

Or rather, I've never seen version requirements that have no minimum version by do have a maximum version (though I'm sure they exist somewhere)